plugin/lspconfig.lua

```diff
@@ -0,0 +1,85 @@
+local version_info = vim.version()
+if vim.fn.has 'nvim-0.7' ~= 1 then
+  local warning_str = string.format(
+    '[lspconfig] requires neovim 0.7 or later. Detected neovim version: 0.%s.%s',
+    version_info.minor,
+    version_info.patch
+  )
+  vim.notify_once(warning_str)
+  return
+end
+
+local util = require 'lspconfig.util'
+local configs = require 'lspconfig.configs'
```

Comment:
Inconsistent use, sometimes these local variables are used, sometimes `require` is called again. In my opinion you should inline these and always call `require`, to not unnecessarily force these modules to be loaded.

Reply:
`configs` is used several times in the file. Perhaps I only inline `utils`?

Reply:
It's just three times basically.